listeners.py

```diff
@@ -70,7 +70,7 @@ def get_long_mark(self, task):
         return '(long)' if task in self.long_tasks else ''
 
     def prettify_task_name(self, task_id):
-        return task_id[0] + ((':' + task_id[1]) if task_id[1] else '')
+        return task_id[0] + (('-' + task_id[1]) if task_id[1] else '')
```

Comment:
I don't think that we should change `test:conf` format shown on the screen.

Reply:
Yep, indeed, it will be easier to read this way.

Removed these changes.